tests/services/logging.service.spec.ts

```diff
@@ -1,64 +1,130 @@
 import { LoggingService } from '../../src/app/core/services/logging.service';
+import { MockLocalStorage } from '../helper/mock.local.storage';
 
 let loggingService: LoggingService;
-const getFilteredLogCount: (currentLog: string, predicate: (line: string) => boolean) => number = (
-  currentLog: string,
-  predicate: (line: string) => boolean
-) =>
-  loggingService
-    .getTrimmedLogCache(currentLog, loggingService.LOG_COUNT_LIMIT)
-    .split('\n')
-    .filter((line: string) => predicate(line)).length;
-const oldLogIdentifier = 'Old Log';
-const repeatOldLogStartHeader: (numberOfRepitions: number) => string = (numberofRepitions: number) => {
-  return `${loggingService.LOG_START_HEADER}\n${oldLogIdentifier}\n`.repeat(numberofRepitions);
+let headerLog: string;
+let sessionSeparator: string;
+const mockDate = new Date(2021, 6, 27);
+const infoLogMessage = 'Info log message';
+
+const testAddLog = (message: string) => {
```

Comment:
@anubh-v 
It's because before the review changes, I had multiple similar `adding_logs` tests and use this function to reduce code duplication. We can move this now into the test case directly. Thanks for the catch!